raiden/transfer/channel.py

```diff
@@ -534,14 +535,19 @@ def get_status(channel_state):
     return result
 
 
-def del_lock(end_state, hashlock):
-    assert is_known(end_state, hashlock)
+def _del_lock(end_state, hashlock):
```

Comment:
Why add the underscore prefix to the `del_lock()` function?

Reply:
It must be considered private to the module